generators/entity-server/templates/src/main/resources/config/liquibase/changelog/added_entity.xml.ejs

```diff
@@ -159,11 +161,11 @@
           Spring Boot configuration key.
     -->
     <changeSet id="<%= changelogDate %>-1-data" author="jhipster" context="faker">
-        <loadData
+        <loadUpdateData
```

Comment:
This is required because of https://liquibase.jira.com/browse/CORE-3208 : `type="computed"` doesn't work currently with `loadData`